frontend/magic_links/templates/landing.html

```diff
@@ -74,7 +74,7 @@ <h2 class="govuk-heading-m">
     <p>
         {% if link_to_contact_us_page %}
             {% if show_contact_us %}
-                If you need help, <a class="govuk-link" href="{{ contact_us_url }}">contact us.</a>
+            {% trans %}If you need help,{% endtrans %} <a class="govuk-link" href="{{ contact_us_url }}">{% trans %}contact us.{% endtrans %}</a>
             {% else %}
                 If you have questions about the form or the fund, <a class="govuk-link" href="{{ contact_us_url }}">contact us.</a>
```

Comment:
Shouldn't this bit have a `{% trans %}` tag as well? The page below does

Reply:
thanks for flagging that, I have added the {% trans %} tag around it.